HIEUNTK24406H/baitapbuoi4/cau6

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    int nam;
+    cout << "nhap vao nam bat ky : ";
+    cin >> nam;
+    if (nam % 4 == 0 and nam % 100 != 0)
```

Comment:
dùng 'and' cũng đc mà nên dùng && vì đa số đều dùng và dễ đọc.
Bài này thiếu 1 trường hợp (điều kiện), năm 2000 là năm nhuận nhưng kết quả chương trình sai